src/FSharp.Data.DesignTime/paket.references

```diff
@@ -0,0 +1,8 @@
+FSharp.Core framework: netstandard2.0
```

Comment:
we can use the `framework: xxxx` syntax to use particular versions of FSharp.Core (or any other package) for a particular framework. Here we use 4.3.4 for netstandard2.0, and 4.0.0.1 for net462.